### ISSUE TITLE
gravatar.com 

### ISSUE THREAD

**Issue by MayaBi7** [lightswitch05/hosts#389]
Can u unblock this site...
     Virustotal Scan Result - https://www.virustotal.com/gui/url/9d70cd59458f93f36804a7f4f58a018a0cff28490eca30bca4ed82689cbdf495/summary

**Comment by iamtalhaasghar** [comment]
Yes, this site does not contains any ads. This break's functionality on jira. User gravatars do not work if this site is blocked.
